src/gluon/services/team/TeamSlackChannelService.ts

```diff
@@ -47,6 +48,7 @@ export class TeamSlackChannelService {
 
     public async addSlackDetailsToGluonTeam(gluonTeamId: string,
                                             slackChannelName: string,
+                                            memberId,
```

Comment:
Please add a type here
```suggestion
                                            memberId: string,
```